### PR TITLE
Opitimize the pull request template

### PR DESCRIPTION
**Pull request by afc163**
### 🧐 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 What's the background and how to resolve it?

根据实践简化 pull request 模板，使其更美观实用。

### 📝 Changelog description

No any.

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Please makes sure that these form are filled before submitting your pull request
 
 [[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]
 
-### This is a ...
+### 🧐 This is a ...
```
Comment (zy410419243) [inline .github/PULL_REQUEST_TEMPLATE.md]: 大概是多了个空格，windows 下看是这样的
p f 2_ ob e exs0 0gr0w

File: .github/PULL_REQUEST_TEMPLATE/pr_cn.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+首先，感谢你的贡献！😄
 
 新特性请提交至 feature 分支，其余可提交至 master 分支。
 在一个维护者审核通过后合并。
 请确保填写以下 pull request 的信息，谢谢！~
 
 [[English Template / 英文模板](?expand=1)]
 
-### 这个变动的性质是
+### 🧐 这个变动的性质是？
```
Comment (zy410419243) [inline .github/PULL_REQUEST_TEMPLATE/pr_cn.md]: 这里也一样

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (afc163) [reply .github/PULL_REQUEST_TEMPLATE.md]: 看来是 emoji 不支持。

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -18,36 +18,22 @@ Please makes sure that these form are filled before submitting your pull request
...
 
-### What's the background?
+### 💡 What's the background and how to resolve it?
 
-> 1. Describe the source of requirement.
-> 2. Resolve what problem.
-> 3. Related issue link.
+> 1. Describe the source of requirement, like related issue link.
+> 2. Describe the existed solution.
+> 3. List final API realization and usage sample.
```
Comment (zombieJ) [inline .github/PULL_REQUEST_TEMPLATE.md]: 这个还是推荐拆开，否则容易漏……

**Pull request merged**